backend/inspirehep/submissions/marshmallow/job.py

```diff
@@ -90,7 +90,8 @@ def build_job(self, data):
         for region in data.get("regions", []):
             job_builder.add_region(region)
         for arxiv in data.get("field_of_interest", []):
-            job_builder.add_arxiv_category(arxiv)
+            if arxiv != "other":
```

Comment:
not needed https://github.com/inspirehep/inspire-schemas/blob/ac80aca588c099e1c01393fae41025f754250d50/inspire_schemas/builders/jobs.py#L194